guides/common/modules/proc_adding-images-to-server.adoc

```diff
@@ -30,7 +30,13 @@ ifdef::kvm-provisioning[]
 endif::[]
 ifdef::rhv-provisioning,openstack-provisioning,gce-provisioning[. From the *Image* list, select an image from the {CRname} compute resource.]
 ifdef::azure-provisioning[]
-. In the *Marketplace Image URN* field, enter a Marketplace URN. For example, *RedHat:RHEL:7-RAW:latest*. For more information, see https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage[Find Linux VM images in the Azure Marketplace with the Azure CLI].
+. In the *Azure Image Name* field, enter a URN in the format `_prefix://_URN`.
```

Comment:
Can we say `Image Name or Marketplace URN` here instead of just `URN`? Since URN is the term used for public/marketplace images.

Reply:
Done.